core/framework/src/main/java/org/phoebus/framework/adapter/AdapterService.java

```diff
@@ -88,27 +88,27 @@ public static List<AdapterFactory> getAdaptersforAdaptable(Class cls)
     }
 
     /**
-     * Adapts the adaptableObject to type adapterType using registered adaptor factories.
+     * Adapts the adaptableObject to type targetType using registered adaptor factories.
      * @param <T> , the type to convert
      * @param adaptableObject , the object to adapt
-     * @param adapterType , the target type 
-     * @return an {@link Optional} with the the adapted object or empty
+     * @param targetType , the target type of <code>adaptableObject</code>
+     * @return an {@link Optional} with the adapted object or empty
      */
-    public static <T> Optional<T> adapt(Object adaptableObject, Class<T> adapterType)
+    public static <T> Optional<T> adapt(Object adaptableObject, Class<T> targetType)
```

Comment:
`targetType` is indeed a good choice, makes the meaning of that parameter more obvious

Reply:
Yes, I found the API a bit unclear precisely for this reason.